### ISSUE TITLE
restrictions for feedback

### ISSUE THREAD

**Issue by paszin** [paszin/goalden#12]
- only submit feedback once per player
- no self feedback

We have to update our datamodel to store who gave feedback.

Is that necessary for the MVP Milestone?

**Comment by clarkjung** [comment]
I don't think this is necessary for the MVP. Let's get our MVP done as soon as possible, and discuss updating our datamodel.

**Comment by paszin** [comment]
I agree. 
But I will implement the no self feedback rule in the frontend

**Comment by clarkjung** [comment]
Great!
